tests/4N-coronene/PPSTM_simple.py

```diff
@@ -156,6 +156,7 @@
     extent = (x[0],x[1],y[0],y[1])
     tip_r  = RS.mkSpaceGrid(x[0],x[1],x[2],y[0],y[1],y[2],z[0],z[1],z[2])
     lvec   = np.array([[x[0],y[0],z[0]],[x[1]-x[0],0.,0.],[0.,y[1]-y[0],0.],[0.,0.,z[1]-z[0]]])
+    atomic_info_or_head = (None, None)
```

Comment:
I would put there just `None` here, because of this:
https://github.com/Probe-Particle/ppafm/blob/6f741fcc1e77de55b3bee873aef4106a36192a65/ppafm/io.py#L857 
We could also change it with the XSF directly at PPAFM.

Reply:
Hmm, I'm not sure if I see the problem. In `PPSTM_simple.py`, `atomic_head_or_info` is only used here in the  saving procedure https://github.com/Probe-Particle/PPSTM/blob/82a977aa828adf1b316806c9cd6c1a68741b5ddb/PPSTM_simple.py#L370-L379

The actual saving function [ppafm.io.saveNpy](https://github.com/Probe-Particle/ppafm/blob/6f741fcc1e77de55b3bee873aef4106a36192a65/ppafm/io.py#L730C1-L740C95) then requires a tuple as input. I don't think that `io.save_vec_field` is used here. Is there something I'm overlooking?

Reply:
Ok, I got it, my mistake, maybe it should be completely rewritten in the future.